src/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rdoc', '~>4.0'
+gem 'rspec', '~> 3.0'
```

Comment:
Sure, if I'm not mistaken, when you run 
```
bundle install
```
it will install development and runtime dependencies, but if you install gem from rubygems
```
gem install symengine
```
it will install only runtime dependencies.